### ISSUE TITLE
jinja2 requirement not declared, refgenie fails to run

### ISSUE THREAD

**Issue by ialbert** [pepkit/pipestat#122]
I believe that I have tracked a `refgenie` failure to run to this package 

I think it is caused because `jinja2` dependency in not listed as a requirement of this package

Running a fresh install of `refgenie` generates the following error:

```
pipestat/html_reports_pipestat.py", line 9, in <module>
    import jinja2
ModuleNotFoundError: No module named 'jinja2'
```

**Comment by nsheff** [comment]
@ialbert thank you for raising this issue! sorry for the inconvenience

I believe we have solved this on dev: https://github.com/pepkit/pipestat/issues/91

@donaldcampbelljr we should probably do a bugfix release to fix this ASAP, since it's breaking other tools (refgenie), unless the release of full new pipestat version is imminent

**Comment by ialbert** [comment]
I feel that the bug is a showstopper for anyone not sufficiently well-versed in Python packaging - basically renders the refgenie tool unusable.

The conda versions do work as expected - but now that I looked more closely the same refgenie 0.12.1 installed via conda does not the use pipestat library - which I found quite suprising

upon even more investigation, it looks like a package called piper that started to use pipestat with a certain version.  The conda version being in a working state is more of an accidental outcome of the various package resolution orders. But basically what this means that piper does not work either on a fresh install on the same account. Even more reason to add the bugfix.

With tools like refgenie I would strongly suggest that the documentation recommends installation via  `pipx` - rather than `pip`  the tool has a large number of dependencies with unexpected resolutions.
